src/StochasticDiffEq.jl

```diff
@@ -20,7 +20,7 @@ module StochasticDiffEq
 
   import DiffEqBase: resize!,deleteat!,addat!,full_cache,user_cache, u_cache,du_cache,
                      resize_non_user_cache!,deleteat_non_user_cache!,addat_non_user_cache!,
-                     terminate!,get_du, get_dt,get_proposed_dt,
+                     terminate!,DiffEqDiffTools.get_du, get_dt,get_proposed_dt,
```

Comment:
Don't shadow here. This is a "different `get_du`" which is part of the integrator interface.